src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplAPISteps.java

```diff
@@ -24,4 +34,56 @@ public void verifyContentUnderCertifiedHealthITProductListingAndURL(final String
         assertEquals(actualText, text);
         assertEquals(actualURL, urlLink);
     }
+
+    /**
+     * Assert that title should be CHPL API.
+     * @param title expected as CHPL API
+     */
+    @Then("^CHPL API page title should be \"([^\"]*)\"$")
+    public void chplApiPageTitleShouldBe(final String title) {
+        String chplAPITitle = getDriver().getTitle();
+        Assert.assertEquals(chplAPITitle, title);
+    }
+
+    /**
+     * Click all the controller elements link.
+     * @param arg1 as controller elements
+     */
+    @When("^I click on \"([^\"]*)\" link$")
+    public void iClickOnLink(final String arg1) {
+        String xPath = "//*[text()='" + arg1 + "']";
```

Comment:
Steps shouldn't be using xPaths. Only Page Objects should have xPath stuff. Steps could pass arguments to pages if there are multiple types of things, like in the API lists